.gitignore

```diff
@@ -64,3 +64,4 @@ installation/resources/cluster-certificate-secret.yaml
 installation/resources/ui-test-secret.yaml
 installation/components.env
 installation/versions.env
+installation/versions-overrides.env
```

Comment:
is versions.env still using? Any documentation? Without documentation, it is difficult to do a reliable review.

Reply:
`versions.env` can still be on developers machines so it can't be removed from `.gitignore` yet. File named `versions-overrides.env` was introduced to mitigate impact of existing versions.env file.

Reply:
I also updated the readme describing the versions-override.env file

Reply:
Won't this 2 files coexisting cause confusion?

Reply:
OK, I removed `versions.env` from gitignore, the only valid file is `versions-overrides.env` and in this PR there are changes in CI to produce this file in terms of integration testing. Also readme describes the purpose of this file.